config/application.rb

```diff
@@ -10,7 +10,7 @@
 
 module HotlineWebring
   class Application < Rails::Application
-    config.load_defaults 6.1
+    config.load_defaults 7.0
```

Comment:
Here's what this does: https://guides.rubyonrails.org/configuring.html#results-of-config-load-defaults

Honestly, I don't know what every setting does, but we do so little in this app that if the tests pass, I'm confident that we're fine. We could break cookies and it would be fine.